pkg/console/operator/sync_v400.go

```diff
@@ -593,3 +601,12 @@ func (co *consoleOperator) GetAvailablePlugins(enabledPluginsNames []string) []*
 	}
 	return availablePlugins
 }
+
+func getNodeArchitectures(nodes *corev1.NodeList) []string {
+	nodeArchitectures := sets.NewString()
+	for _, node := range nodes.Items {
+		nodeArch := node.Labels[api.NodeArchitectureLabel]
+		nodeArchitectures.Insert(nodeArch)
+	}
+	return nodeArchitectures.List()
```

Comment:
Will this return the architectures in the same order every time? If not, will it introduce unnecessary rollouts of console?

Reply:
sorry, somehow the sorting was removed in my last update... its back now 👍

Reply:
`List` is already ordered, fwiw. https://pkg.go.dev/k8s.io/apimachinery/pkg/util/sets#String.List